mspray/apps/reactive/irs/utils.py

```diff
@@ -0,0 +1,36 @@
+"""Utils module"""
+from django.conf import settings
+from django.contrib.gis.geos import MultiPolygon
+
+from mspray.apps.main.models import Location
+
+
+def get_chw_location(chw: object):
+    """
+    Gets or creates a Community Health Worker location
+
+    :param chw: Community Health Worker object
+    :return: Location object linked to the Community Health Worker
+    """
+
+    level = settings.MSPRAY_REACTIVE_IRS_CHW_LOCATION_LEVEL
```

Comment:
This puts a hard requirement on the setting `MSPRAY_REACTIVE_IRS_CHW_LOCATION_LEVEL` to be defined. Could we use `getattr()` with a default value instead?

Reply:
will do!

Reply:
will do